.gitlab-ci.yml

```diff
@@ -107,6 +107,22 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
 
+.requires-docker: &requires-docker
```

Comment:
Nit pick, this should be next to the other yaml anchor `&dind-login`.

I think that I like more your porposal of having it after the `include`, but then both should be here.

Reply:
sure, no worries, moved around I hope I have not lost anything.